modules/database/src/ioc/as/asIocRegister.c

```diff
@@ -78,8 +86,12 @@ static void asphagCallFunc(const iocshArgBuf *args)
 /* asprules */
 static const iocshArg asprulesArg0 = { "asgname",iocshArgString};
 static const iocshArg * const asprulesArgs[] = {&asprulesArg0};
-static const iocshFuncDef asprulesFuncDef = {"asprules",1,asprulesArgs,
-                                             "List rules of an Access Security Group.\n"};
+static const iocshFuncDef asprulesFuncDef = {
+    "asprules",1,asprulesArgs,
+    "List rules of an Access Security Group.\n"
+    "If no Group is speciﬁed then list the rules for all groups\n"
+    "Example: asgname mygroup"
```

Comment:
Should be

> Example: asprules mygroup

Reply:
Resolved in 7370bba0c841d6564a735bc6d57297a3b7e43e3c